marquee/marquee.ino

```diff
@@ -1465,7 +1465,7 @@ void centerPrint(String msg, boolean extraStuff) {
   // Print the static portions of the display before the main Message
   if (extraStuff) {
     if (!IS_24HOUR && IS_PM && isPM()) {
-      matrix.drawPixel(matrix.width() - 1, 6, HIGH);
```

Comment:
I actually like it on the edge.   I am good with it.  You ok if we keep it at the edge?

Reply:
I added a option to select your corner, after looking at clocks all weekend, they are all over the place, so I added lots of options... See the latest commit.

Thank you.